src/main.cpp

```diff
@@ -708,9 +708,11 @@ void setup()
     homeKeyReader::readerData_t p = data.template get<homeKeyReader::readerData_t>();
     readerData = p;
   }
-  // homeSpan.setStatusPin(2);
-  // homeSpan.setStatusAutoOff(5);
+  homeSpan.setWifiCredentials(SSID, WIFI_PWD);
```

Comment:
think would be better not to hardcode the wifi credentials, instead we can give the option to assign a control pin using `homespan.setControlPin` to enable Homespan's Device Configuration Mode